### DISCUSSION TITLE
希望加入服务器中转功能

### DISCUSSION THREAD

**Discussion by 6396036** [AlistGo/alist#325] [Q&A] [answered]
实话讲，这是我用过最棒的目录程序了。
只是有个建议，不知道大佬能否支持。
国外大盘鸡直连速度较慢，香港日本等VPS直连好但是硬盘小。
希望加入服务器中转功能，这样就可以用香港日本小鸡，通过FTP方式从大盘鸡上拖文件。
现在FTP模式没有proxy（代理）选项，是不是表示不支持中转。

**Comment by xhofe** [comment, marked as answer]
FTP默认就是用服务器中转的 没法直连，所以没有proxy的选项。

**Reply by 6396036** [reply]
我单独从服务器上下载是可以跑满的，可通过服务拖文件反而比从原大盘鸡上更慢，可能是我哪里没设置对，求大佬指导

**Reply by xhofe** [reply]
这个我也不太清楚啊，可能是FTP协议比较慢？我访问本机的FTP都感觉很慢。。。

**Reply by 6396036** [reply]
大佬，针对这种 中转拖文件，有没有什么使用建议？
